lib/std/zig/parser_test.zig

```diff
@@ -4,6 +4,22 @@
 // The MIT license requires this copyright notice to be included in all copies
 // and substantial portions of the software.
 
+test "zig fmt: comment or multiline string in field decl w/ no trailing comma" {
+    try testCanonical(
+        \\const S = struct {
+        \\    a: void, // a
+        \\    b: void
```

Comment:
We should insert a trailing comma if there is not one present in this case and use `testTransform()` to check that this is done. This would be consistent with the handling of trailing commas in struct/array initializers and function arguments. 
```suggestion
        \\    b: void,
```